docs/en/admins/03_Installation.md

```diff
@@ -10,14 +10,16 @@ Before you begin, make sure that you've read the [prerequisites](02_Prerequisite
 
 3. Give ownership of the FreshRSS folder to your web server user (often `www-data`). Give group read permissions to all files in `.`[^2], and group write permissions to `./data/`.
 
-4. Install needed PHP modules.
+4. Install needed PHP modules: libxml, cURL, JSON, PDO_MySQL, PCRE, and ctype. (Zlib, mbstring, iconv, and ZipArchive are recommended as well).
```

Comment:
We can refer to our Dockerfile for Debian and Alpine for a precise and up-to-date list:
* https://github.com/FreshRSS/FreshRSS/blob/master/Docker/Dockerfile#L11-L12
* https://github.com/FreshRSS/FreshRSS/blob/master/Docker/Dockerfile-Alpine#L7-L9